src/__tests__/AppTests.js

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from "@testing-library/react";
+import App from "../App";
+
+test("demo", () => {
```

Comment:
Why we need this test case

Reply:
Its just a demo test. for test jest library.